src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -202,19 +198,28 @@ public void WhenNewIssuesAreFound_ButForIncorrectFile_ListenersAreNotUpdated()
             // Assert
             CheckSinkNotNotified(errorListSinkMock1);
             CheckSinkNotNotified(errorListSinkMock2);
+
+            issuesFilter.Verify(x => x.Filter(It.IsAny<string>(), It.IsAny<IEnumerable<IFilterableIssue>>()), Times.Never);
         }
 
         [TestMethod]
-        public void WhenNewIssuesAreFound_ListenersAreUpdated()
+        public void WhenNewIssuesAreFound_OutOfRangeIssuesAreIgnored_ListenersAreUpdated()
         {
             // Arrange
-            var issues = new[] {
-                new Sonarlint.Issue { RuleKey = "S111", StartLine = 1, EndLine = 1 },
+            var issues = new[]
+            {
+                new Issue { RuleKey = "S111", StartLine = 1, EndLine = 1 },
                 // The next issue is outside the range of the new snapshot and should be ignored
-                new Sonarlint.Issue { RuleKey = "S222", StartLine = 99999998, EndLine = 99999999 },
-                new Sonarlint.Issue { RuleKey = "S333", StartLine = 100, EndLine = 101 }
+                new Issue { RuleKey = "S222", StartLine = 99999998, EndLine = 99999999 },
+                new Issue { RuleKey = "S333", StartLine = 100, EndLine = 101 }
             };
 
+            // Setup up the filter to return whatever was supplied
+            var originalIssues = new List<IFilterableIssue>();
```

Comment:
Maybe call SetupIssuesFilter? It can receive optional output param, that if not specified, returns the original list.